modules/administration/pages/virtual-hosts.adoc

```diff
@@ -111,4 +111,10 @@ If you do not refresh the data from a new Virtual Host Manager, host data will n
 ====
 +
 
-. View inventoried ESXi host objects and resources by selecting menu:Systems[Virtual Systems] .
+. View inventoried ESXi host objects and resources by selecting menu:Systems[Virtual Systems].
```

Comment:
You changes the second level "Systems" to "System List" and you did not tell us. :unamused: Thus we not either must use as a short version:

```suggestion
. View inventoried ESXi host objects and resources by selecting menu:System List[Virtual Systems].
```

Or the long version:
```
menu:Main Menu[Systems > System List > Virtual Systems]
```
But do not worry too much. There are inconsistencies with marking up menu entries everywhere...  And I do not know what we can do about it.

Reply:
> 
> Or this long version:
> 
> ```
> menu:Main Menu[Systems > System List > Virtual Systems]
> ```
> 
I dislike the "main menu" thing, it just makes it laborious (to write and to read).

> But do not worry too much. There are inconsistencies with marking up menu entries everywhere... And I do not know what we can do about it.

Do a consistency sweep and update them all? We need to do this for a few things, I think ... (post 4.0 jobs!)